bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -129,6 +129,12 @@ impl LockTime {
 
     /// Constructs a `LockTime` from `n`, expecting `n` to be a valid block height.
     ///
+    /// ## Note on Locktime and Block Inclusion:
```

Comment:
Aren't we using only H1 now since #2832?
cc @jamillambert

Reply:
Yeah, I guess we should stick with H1 even though this is a "minor" thing. I guess the heading could be shortened to just `# Note` so that the reader isn't hit with a ton of big text.

Reply:
Ok , I'll change it.
Thanks for pointing the mistake.